wrappers/here.rb

```diff
@@ -463,5 +478,9 @@ def max_srcs(dist_km)
         [25 - (dist_km / 100).floor, 1].max
       end
     end
+
+    def trace(message)
```

Comment:
why do not use standard `level` param/way to display or not traces depending environment?
- in local/dev: tracel level could be "debug"
- in production: trace level is "info"

Reply:
This traces can be very verbose, I did not want to log in production mode. By the way what do you mean by *standard level param/way* we keep the method but depending on environment we trace in debug or info ?

Reply:
I have moved all traces to `Api::Root.logger.debug`.